judge/models/runtime.py

```diff
@@ -147,6 +149,13 @@ def disconnect(self, force=False):
 
     disconnect.alters_data = True
 
+    def toggle_disabled(self):
+        self.is_disabled = not self.is_disabled
+        self.save(update_fields=['is_disabled'])
+        disconnect_judge(self, force=True)
```

Comment:
Should this send a forced disconnect if you're re-enabling the judge?

In particular, it looks like this function is used less like *toggle* disable, and more like *disable*.

Reply:
The purpose of forcing a disconnect is to update the `is_disabled` cached property in the bridge, which helps to not access the Judge DB object for every submission and queue removal.

Reply:
I think there should be a bridge RPC added rather than hacking around it this way. The judge we run on dmoj.ca will auto-reconnect, but it's not guaranteed that all judges will.

Reply:
Added a bridge RPC instead of the disconnect method.